model/miner_ask.go

```diff
@@ -1,28 +1,19 @@
 package model
 
 import (
+	"github.com/application-research/estuary/config"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-state-types/big"
 	"github.com/filecoin-project/lotus/chain/types"
 	"gorm.io/gorm"
 )
 
-var priceMax abi.TokenAmount
-
-func init() {
```

Comment:
removed this has it is already set for config at bootup here https://github.com/application-research/estuary/blob/dev/constants/constants.go#L40